MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/meta.json

```diff
@@ -1,10 +1,10 @@
 {
-  "risk_rating": "info",
+  "risk_rating": "secure",
   "short_description": "VirusTotal Malware analysis.",
   "references": {
     "Virustotal": "https://www.virustotal.com/"
   },
-  "title": "Virustotal malware analysis (MD5 based search)",
+  "title": "Secure Virustotal malware analysis (MD5 based search)",
   "privacy_issue": true,
   "security_issue": true
```

Comment:
The `privacy_issue` & `security_issue` should be set to false in the secure case, no?